spec/support/shared_context/rake_helper.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+# https://robots.thoughtbot.com/test-rake-tasks-like-a-boss
+require "rake"
+
+shared_context "rake" do
```

Comment:
I will rename the file to `rake_helper_spec`, this way it shouldn't be consider by SimpleCov. If it doesn't change anything, I will exclude the file manually.